api/app/Providers/AuthServiceProvider.php

```diff
@@ -109,7 +109,7 @@ public function resolveUserOrAbort($bearerToken, $tokenService): ?User {
 
         // If a default user is set, and there is no bearer token, treat the request as if it were from the default user.
         if (config('oauth.default_user')) {
-            $user = User::where('email', config('oauth.default_user'))->first();
+            $user = User::where('sub', config('oauth.default_user'))->first();
```

Comment:
I'm not sure if this is an improvement? Since this setting is only for convenience during local development, I think using an email is more intuitive. I can set `AUTH_DEFAULT_USER=admin@test.com` without having to look up the sub value.

Reply:
I changed this because it is the last remaining spot that `email` is used in an auth context so it is very out of place.  The seeder also seeds "admin@test.com" as the `sub` for this user so you don't have to look up a new value - it's the same. :smile: